lib/elixir/lib/enum.ex

```diff
@@ -764,15 +775,15 @@ defmodule Enum do
   end
 
   defp fetch_enumerable(enumerable, index, module) do
-    reduce_result =
+    reduce_res =
```

Comment:
Sorry It got messed up, probably because I based these changes on code previous to @lexmag correction.
change reverted, all good then